hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducerListener.java

```diff
@@ -482,6 +482,6 @@ public Throwable getCause() {
     }
 
     public enum Status {
-        SUCCESS, FAIL
+        SUCCESS, FAIL, SKIP
```

Comment:
Hum, is SKIP specific to validation?  Or is it also applicable to general HollowProducerListeners?  In that case, does it mean that the cycle was skipped?

Reply:
My thinking is that indicating skipped status when possible provides richer information. For example, if there are no changes publish, announce, validate phases could certainly say skipped.

Reply:
Are the code changes done to use SKIP when there is nothing to publish/announce?

Reply:
No I have not modified other Hollow cycle stages to use Skip. Those changes can be made as needed when working in the area.

Reply:
Might be good idea to add it when there is an use case for it if there is none yet.